dace/config_schema.yml

```diff
@@ -770,6 +770,184 @@ required:
+                        title: Local matlab solver
+                        default: ""
+                        description: >
+                            Path to local Matlab solver's launch file.
+
+                    remote_solver_address:
+                        type: string
+                        title: Remote matlab solver
+                        default: ""
+                        description: >
+                            Address of the remote Matlab server.
+
+                    only_db:
+                        type: bool
+                        title: Turns off Matlab solver. Relying only on precomputed solutions
+                        default: true
+                        description: >
+                            If only_db is specified, only the problems already analyzed in the
+                            solver's database can be processed. If new problems are encountered,
+                            the solver throws an error.
+
+                    remote_matlab:
+                        type: bool
+                        title: Using remote matlab solver
+                        default: true
+                        description: >
+                            If not relying on the solver's database (solver_only_database),
+                            it specifies whether one uses the local or remote Matlab server
+
+                    caching_solver_solutions:
+                        type: bool
+                        title: Caching newly computed solver results
+                        default: true
+                        description: >
+                            If matlab solver is used and problems that are not in the solver's database 
+                            are encountered, it updates the database with the new solutions.
+
+
+            # output generation
+            output:
+                type: dict
+                title: Output
+                descripion: Output format parameters
+                required:
+                    latex:
+                        type: bool
+                        title: Output in a latex format
+                        default: false
+                        description: >
+                            The I/O analysis output can be formatted as a latex table
+
+                    only_leading_term:
+                        type: bool
+                        title: Output only asymptotically leading monomial
+                        default: false
+                        description: >
+                            Output only asymptotically leading monomial
+
+                    all_params_equal:
+                        type: bool
+                        title: Simplified output result
+                        default: false
+                        description: >
+                            To simplify the output, all params can be assumed equal (e.g., N^3 instead of N*M*K)
+
+
+            # tests 
+            tests:
```

Comment:
I am not really a fan of having configuration parameters for tests but I also see the issue with needs SDFGs that are not included in a DaCe installation by default. We should discuss.